test/disable-goosig-test.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * test/disable-goosig-test.js - Test disabling GooSig
+ * Copyright (c) 2020, The Handshake Developers (MIT License).
+ * https://github.com/handshake-org/hsd
+ */
+
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const fs = require('fs');
+const {resolve} = require('path');
+const FullNode = require('../lib/node/fullnode');
+const AirdropProof = require('../lib/primitives/airdropproof');
+const BlockTemplate = require('../lib/mining/template');
+const consensus = require('../lib/protocol/consensus');
+
+// Use a valid, working airdrop proof
+const AIRDROP_PROOF_FILE = resolve(__dirname, 'data', 'airdrop-proof.base64');
+const rawAirdropProof =
+  Buffer.from(fs.readFileSync(AIRDROP_PROOF_FILE, 'binary'), 'base64');
+const proof = AirdropProof.decode(rawAirdropProof);
+
+let node, networkAirdrop;
+describe('Disable GooSig', function() {
+  describe('Before Disable', () => {
+    before(async () => {
+      node = newNode();
+      await node.open();
+    });
+
+    after(async () => {
+      await node.close();
+      node.network.airdrop = networkAirdrop;
+    });
+
+    it('should accept GooSig based airdrop in mempool before', async () => {
```

Comment:
nit: do we need "before" at the end of this description? The wrapping describe block implies this. This nit applies across all `it` blocks.